BSDmakefile

```diff
@@ -0,0 +1,5 @@
+default:
```

Comment:
Not followig this very closely, but it's unfortunate to have to clutter the top level dir with this